float64[] goal
float64[] position
float64[] velocity
float64[] command
string[] ext_name
float64[] ext_value
